content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+# Announcing scip-clang: a new SCIP indexer for C and C++
+
+[scip-clang](https://github.com/sourcegraph/scip-clang) is our new indexer
+for C and C++ code written from the ground up to natively emit [SCIP](https://github.com/sourcegraph/scip)
+and especially support the wide range of language features present in C++.
+
+scip-clang is now available in beta, and replaces our older indexer lsif-clang.
```

Comment:
Expand this for a bit of rationale as to why we're replacing lsif-clang.

Reply:
Isn't this immediately covered by the points about improvements over lsif-clang?

Reply:
With my audience hat on not knowing as much context, I'd wonder why those improvements weren't just done in lsif-clang?

Reply:
That will be answered in the separate blog post. https://sourcegraph.slack.com/archives/CHXHX7XAS/p1679578225074519